### ISSUE TITLE
Running The Broken Font troubleshooter on Windows 11 Version 22H2v2 or higher mentions it is going to be retired

### ISSUE THREAD

**Issue by gus33000** [File-New-Project/TroubleshootingPacks#1]
# Problem

When running the broken font troubleshooter on Windows 11 Version 22H2v2 or higher the following notice is displayed:

![image](https://github.com/File-New-Project/TroubleshootingPacks/assets/3755345/f5bed5fb-8030-4386-9809-664f74f2d13d)

According to the page linked on such message, the troubleshooter is going to be retired by Microsoft, following a specific deprecation schedule (more here: https://support.microsoft.com/en-us/windows/deprecation-of-microsoft-support-diagnostic-tool-msdt-and-msdt-troubleshooters-0c5ac9a2-1600-4539-b9d0-069e71f9040a)

# Expected behavior

The ability to fix the problem should remain in place whenever or not MSDT is going away, or not.

**Comment by riverar** [comment]
Yup, good eye. On it already, filed a similar issue in feedback hub last week https://aka.ms/AAkx50k.

> Title: Troubleshooter deprecation plan doesn't outline impact to third-party troubleshooters
> The support page doesn't make clear what will happen to third-party troubleshooters. [0] (We have one. [1]) It sounds like it will continue to work until 2025? Also a concern is that your sunset warning message is appearing on our troubleshooter pack.

**Comment by gus33000** [comment]
Hi, is there any news on this issue @riverar ?

**Comment by riverar** [comment]
Yes. We will adjust EarTrumpet and discontinue use of this pack. 🐖
